flake-module.nix

```diff
@@ -14,12 +14,12 @@ in {
           that you can reference them in this or another flake's `homeConfigurations`.
         '';
       };
-      homeManagerModules = mkOption {
+      homeModules = mkOption {
         type = types.lazyAttrsOf types.deferredModule;
         default = { };
         apply = mapAttrs (k: v: {
-          _class = "homeManager";
-          _file = "${toString moduleLocation}#homeManagerModules.${k}";
+          _class = "home";
```

Comment:
this didn't appear in the originally reverted commit. Is it your change or a revert from another commit. I dont mind either way, as I suspect it affects only a very limited number of people, just wanted to know.

Reply:
There were a few changes to module since the original

Reply:
I don't think it's correct to set the class to `"home"`

Based on others' comments, the class should still be `"homeManager"`

Reply:
The only comment, that I can see, referring to the class mentions we could change it https://github.com/nix-community/home-manager/pull/6392#issuecomment-2634595310 if we wanted it to match the output.